src/main/java/wooteco/subway/line/LineController.java

```diff
@@ -0,0 +1,71 @@
+package wooteco.subway.line;
+
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import wooteco.subway.exception.LineNameDuplicatedException;
+import wooteco.subway.exception.LineNotFoundException;
+import wooteco.subway.line.dao.LineDao;
+
+import java.net.URI;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@RestController
+public class LineController {
+
+    private final LineDao lineDao;
+
+    public LineController(LineDao lineDao) {
+        this.lineDao = lineDao;
+    }
+
+    @PostMapping("/lines")
+    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
+        final String name = lineRequest.getName();
+        final String color = lineRequest.getColor();
+        if (lineDao.findLineByName(name).isPresent()) {
+            return ResponseEntity.badRequest().build();
+        }
+        Line line = new Line(name, color);
+        Line createdLine = lineDao.save(line);
+        LineResponse lineResponse = new LineResponse(createdLine);
+        return ResponseEntity.created(URI.create("/lines/" + createdLine.getId()))
+            .body(lineResponse);
+    }
+
+    @GetMapping(value = "/lines", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<LineResponse>> showLines() {
+        List<Line> lines = lineDao.findAll();
+        List<LineResponse> lineResponses = lines.stream()
+            .map(LineResponse::new)
+            .collect(Collectors.toList());
+        return ResponseEntity.ok().body(lineResponses);
+    }
+
+    @GetMapping(value = "/lines/{id}")
+    public ResponseEntity<LineResponse> findLineById(@PathVariable Long id) {
+        Line line = lineDao.findLineById(id).orElseThrow(LineNotFoundException::new);
+        return ResponseEntity.ok(new LineResponse(line));
+    }
+
+    @PutMapping(value = "/lines/{id}")
+    public ResponseEntity<Void>updateLine(@PathVariable Long id, @RequestBody LineRequest lineRequest) {
```

Comment:
```suggestion
    public ResponseEntity<Void> updateLine(@PathVariable Long id, @RequestBody LineRequest lineRequest) {
```

[Reformat and rearrange code](https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html)
단축키 사용을 습관화하거나, 인텔리제이에서 커밋하신다면 "Reformat code" 옵션을 켜는 건 어떨까요?